### PR TITLE
Gives solemn eulogy an ability

### PR DESCRIPTION
**Pull request by Litberries**
## About The Pull Request

Just makes the guns do more damage with realized

## Why It's Good For The Game

More realization abilities

## Changelog
:cl:

tweak: tweaked a few things

/:cl:

### CODE REVIEW

File: code/modules/clothing/suits/ego_gear/realized.dm [hunk 1]
```diff
@@ -174,7 +174,7 @@ No Ability	250
 	name = "solemn eulogy"
 	desc = "Death is not extinguishing the light, it is putting out the lamp as dawn has come."
 	icon_state = "eulogy"
-	armor = list(RED_DAMAGE = 40, WHITE_DAMAGE = 80, BLACK_DAMAGE = 80, PALE_DAMAGE = 50)		//No ability
+	armor = list(RED_DAMAGE = 40, WHITE_DAMAGE = 70, BLACK_DAMAGE = 70, PALE_DAMAGE = 30)		//Ranged
```
Comment (Kitsunemitsu) [inline code/modules/clothing/suits/ego_gear/realized.dm:177]: We actually are now rebalancing all realized armor to 230.

File: code/modules/projectiles/guns/ego_gun/waw.dm [hunk 2]
```diff
@@ -115,6 +115,17 @@
...
 
+/obj/item/gun/ego_gun/pistol/solemnlament/before_firing(atom/target, mob/user)
+	if(cached_multiplier)
+		projectile_damage_multiplier = cached_multiplier
+	fire_delay = initial(fire_delay)
+	var/mob/living/carbon/human/myman = user
+	var/obj/item/clothing/suit/armor/ego_gear/realization/eulogy/Z = myman.get_item_by_slot(ITEM_SLOT_OCLOTHING)
+	if(istype(Z))
+		cached_multiplier = projectile_damage_multiplier
+		projectile_damage_multiplier *= 2.5
```
Comment (Kitsunemitsu) [inline code/modules/projectiles/guns/ego_gun/waw.dm:126]: This stacks with the 1.5x below. Be advised.

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
Change the armor, the other comment is a warning.

File: code/modules/clothing/suits/ego_gear/realized.dm (hunk 1 above)
Comment (Litberries) [reply code/modules/clothing/suits/ego_gear/realized.dm:177]: so do I re-buff it? it's got 210s because it's ranged

Comment (Kitsunemitsu) [reply code/modules/clothing/suits/ego_gear/realized.dm]: Please do. I'll re-buff it now and then merge.

**Comment by Kitsunemitsu** [comment]
I'm going to rebuff the armor and then merge.

**Review by Kitsunemitsu — APPROVED** [review]
Approved. Looks good to me.

**Pull request merged**